SERVER_CUSTOMIZATION.md

```diff
@@ -135,6 +135,9 @@ The following features are currently configurable:
 * **Line Listing** `LINE_LISTING`: Whether or not using line listing for case entry is enabled in the specified jurisdiction for the specified disease. Configurable from the UI, no database interaction needed.
 * **Documents** `DOCUMENTS`: Enables document storage.
 * **GDPR Popup** `GDPR_CONSENT_POPUP`: Enables a popup which informs new users about the General Data Protection Regulation (GDPR) of the European Union
+* **Task related notifications** `TASK_NOTIFICATIONS`: Enables email and SMS notifications related to tasks management
+* **Confirmed case related notifications** `CONFIRMED_CASE_NOTIFICATIONS`: Enables email and SMS notifications sent to the surveillance officer of events where a participant is a confirmed case
+* **Other notifications** `OTHER_NOTIFICATIONS`: Enables other email and SMS notifications (other than those impacted by `TASK_NOTIFICATIONS` and `CONFIRMED_CASE_NOTIFICATIONS` features)
```

Comment:
CONFIRMED_CASE_NOTIFICATIONS => EVENT_PARTICIPANT_CASE_CONFIRMED_NOTIFICATIONS
Also, remove the "features" word at the end